pkg/jobstore/boltdb/store.go

```diff
@@ -1220,6 +1221,8 @@ func (b *BoltJobStore) updateExecution(tx *bolt.Tx, request jobstore.UpdateExecu
 		}
 	}
 
+	analytics.EmitEvent(context.TODO(), analytics.NewExecutionEvent(newExecution))
+
```

Comment:
we should not publish the event for each execution update. The most valuable event when the execution is terminated as it will give us how long the execution ran, whether it succeeded or not, and reason of failure. Similar to `job.terminal` event.

If you want to publish another event for when the execution was created, then that is fine, but that is mainly going to be useful for long running jobs.

In summary, the must have is `bacalhau.execution_v1.terminal`, and if you want to publish the execution created event, it should be a separate `bacalhau.execution_v1.create`

Reply:
Does this imply we don't want to track executions that are related to bidding?